### DISCUSSION TITLE
Bale Finder

### DISCUSSION THREAD

**Discussion by rento112** [Courseplay/Courseplay_FS22#1063] [Q&A] [answered]
Are we able to select a destination to unload bales or make a course to do that yet? If so, I dont know where to find it

**Comment by Tensuko** [comment, marked as answer]
Unlaod bales is handeld by AutoDrive. Wait for AD Version 2.0.0.5

**Comment by Chico0008** [comment]
Any news regarding the release of AD 2.0.0.5 ? because i'm also very interested on unload bale after picking them in field.
for now ad can drive to a localtion tu "unload" but do not, and make all balls already there messy (because it do not detect balls already there)

**Reply by Tensuko** [reply]
Why do you ask us ?
We are not the AutoDrive developer.
Yes we work with them, yes we talk with them, but we don't ask them about Release plans or something similar.

You need to have patient...
